tests/functional/PendingReason.class.php

```diff
@@ -382,4 +386,59 @@ public function testAddPendingFollowupOnAlreadyPending(): void
         $p_item = PendingReason_Item::getForItem($ticket);
         $this->boolean($p_item)->isFalse();
     }
+
+    public function testRemovePendingAndRevertStatus(): void
```

Comment:
Test is OK but maybe we should test changes/problems too since they are the one that were not behaving properly ?

Reply:
done